app/controllers/timer_sessions_controller.rb

```diff
@@ -1,23 +1,14 @@
 # frozen_string_literal: true
 
 class TimerSessionsController < TrackyController
-  # rubocop:disable Metrics/MethodLength
   def index
     @timer_sessions_in_range = TimerSession.includes(:issues, :time_entries, :timer_session_time_entries)
                                            .finished
                                            .created_by(@current_user)
-    @non_matching_timer_session_ids = TimeDiscrepancyLoader.uneven_timer_sessions(@timer_sessions_in_range).pluck(:id)
-    @timer_sessions = apply_filter(@timer_sessions_in_range, :timer_start)
-    time_entries = time_entries_in_range(@timer_sessions)
-
-    @timer_sessions = (@timer_sessions.order(timer_start: :desc) + time_entries)
-                      .map { |time_entity| TimeEntityDecorator.new(time_entity) }
-                      .sort_by(&:start_time)
-                      .reverse
-                      .group_by(&:entry_date)
+    @non_matching_timer_session_ids = TimeDiscrepancyLoader.uneven_timer_sessions(@timer_sessions_in_range).map(&:id)
```

Comment:
I tried changing the query to use 'pluck' instead of 'map' but unfortunately I couldn't find a way to make it work..